### PR TITLE
Minor CSS fixes in Chats component

### PR DESCRIPTION
**Pull request by madhurg-epns**

### CODE REVIEW

File: src/components/chat/w2wChat/chats/Chats.tsx [hunk 1]
```diff
@@ -335,7 +335,7 @@ const SenderMessage = styled.div`
   box-sizing: border-box;
   position: relative;
   right: 34px;
-  max-width: 419px;
+  max-width: 650px;
```
Comment (mishramonalisha76) [inline src/components/chat/w2wChat/chats/Chats.tsx:338]: this has to be done for receiverMessage box too

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]
image
the received messages width has not increased

File: src/components/chat/w2wChat/chats/Chats.tsx (hunk 1 above)
Comment (madhurg-epns) [reply src/components/chat/w2wChat/chats/Chats.tsx:338]: @mishramonalisha76 I added this change to `ReceivedMessage` as well.

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]
the mobile version isnt right

**Comment by zaryab2000** [comment]
if this is outdated now, we should close it @mishramonalisha76

**Review by corlard3y — COMMENTED** [review]
Screenshot 2023-04-06 at 16 47 13
Experienced this on the PR @madhurg-epns @mishramonalisha76

**Review by corlard3y — APPROVED** [review]

**Review by mishramonalisha76 — APPROVED** [review]

**Pull request merged**